code/modules/SCPmodular/scps/euclid/SCP-173.dm

```diff
@@ -0,0 +1,212 @@
+/mob/living/simple_animal/hostile/scp_173
+	name = "SCP-173"
+	desc = "A statue, constructed from concrete and rebar with traces of Krylon brand spray paint"
+	icon = 'icons/mob/scpicon/scpmobs/scp-173.dmi'
+	ventcrawler = VENTCRAWLER_NUDE
+	icon_state = "173"
+
+	maxHealth = 5000
+	health = 5000
+	move_force = MOVE_FORCE_NORMAL
+	move_resist = MOVE_FORCE_EXTREMELY_STRONG
+	pull_force = MOVE_FORCE_EXTREMELY_STRONG
+	response_help = "touches"
+	response_disarm = "pushes"
+	a_intent = INTENT_HARM
+//	possible_a_intents = list(INTENT_HARM)
+	harm_intent_damage = 1
+	obj_damage = 500
+	melee_damage_lower = 5000
```

Comment:
> instant-killing

That has some pretty serious balance concerns, even if it is accurate to the SCP.

Reply:
this is balanced by being unable to move or interact while being seen, gives enough time for mutiple crew members to whail on it til its dead
being frozen on the spot when in view and (due to blindess spell being broken) not being able to control when you can move or attack, plus having to wait it out anywhere from 25 to 70 seconds to be able to do something is a big nerf, gives someone enough time to run or call for help, and enough time for mutiple crew members to come by and beat on it

Meledamage is irrelevent here as the attacks call for H.death()